### PR TITLE
Display which language codes are currently in use

### PR DESCRIPTION
**Pull request by ZeLonewolf**
This PR adds a small label in the bottom right to display which languages are currently used by the style.

image

### CODE REVIEW

File: src/js/language_label.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    this._map = undefined;
+  }
+}
+
+export var label = new LanguageControl();
+
+export function displayLocales(localeSet) {
+  document.getElementById("language-field").textContent =
+    Array.from(localeSet).join(", ");
+}
```
Comment (1ec5) [inline src/js/language_label.js:35]: `Label.getLocales()` returns an array, not a set. Use [`Intl.DisplayNames`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DisplayNames) to format these IETF language tags as human-readable strings in either the user’s preferred language:

```suggestion
export function displayLocales(locales) {
  let languageNames = new Intl.DisplayNames(locales, {type: "language"});
  let listFormat = new Intl.ListFormat(locales, {type: "disjunction"});
  document.getElementById("language-field").textContent =
    listFormat.format(locales.map(locale => languageNames.of(locale)));
}
```

**Comment by ZeLonewolf** [comment]
Thanks for the human-readable code @1ec5.  Looks awesome!

image

**Review by 1ec5 — APPROVED** [review]

File: src/js/language_label.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  }
+}
+
+export var label = new LanguageControl();
+
+export function displayLocales(locales) {
+  let languageNames = new Intl.DisplayNames(locales, { type: "language" });
+  let listFormat = new Intl.ListFormat(locales, { type: "disjunction" });
+  document.getElementById("language-field").textContent = listFormat.format(
+    locales.map((locale) => languageNames.of(locale))
```
Comment (1ec5) [inline src/js/language_label.js:36]: If you want to link each language to its Wikidata item:

```js
`https://www.wikidata.org/wiki/Special:ItemByTitle/enwiki/ISO%20639:${languageCode}`
```

or to its English Wikipedia article:

```js
`https://en.wikipedia.org/wiki/Special:Search/ISO%20639:${languageCode}`
```

or to its article in the user’s preferred language:

```js
`https://hub.toolforge.org/P305:${locale}`
```

Comment (ZeLonewolf) [reply src/js/language_label.js:36]: Tracked in #611, because I'm not sure which would be the best reference.

**Pull request merged**